rskj-core/src/main/java/co/rsk/mine/gas/provider/MinGasPriceProviderType.java

```diff
@@ -0,0 +1,7 @@
+package co.rsk.mine.gas.provider;
+
+public enum MinGasPriceProviderType {
+    FIXED,
+    WEB,
+    ON_CHAIN
```

Comment:
I know I suggested this, but that was when I thought that the on chain call would be done directly in the vm rather than through eth module. And so, we should call it ETH_CALL.